src/eepp/ui/doc/syntaxdefinitionmanager.cpp

```diff
@@ -1760,6 +1761,7 @@ SyntaxDefinitionManager::SyntaxDefinitionManager() {
 	addVala();
 	addVerilog();
 	addVisualBasic();
+	addSmallBASIC();
```

Comment:
If you look closely you'll see that the `add` calls are sorted by language name (except for plain text), this should go after `addScala();`.